src/types/ui.ts

```diff
@@ -0,0 +1,9 @@
+export interface PanelUI {
```

Comment:
du kan legge inn referanse til denne filen i types/index.ts fila så typen er tilgjengelig for andre

Reply:
done